exp3/TTS_dataset/VAD.py

```
@@ -0,0 +1,243 @@
+                temp_wav_path = temp_file.name
+            
+            # 转换音频格式
+            if not convert_audio_to_wav_format(input_path, temp_wav_path):
+                print(f"  [跳过] 无法转换文件 {os.path.basename(input_path)}")
+                return
+            
+            # 读取转换后的文件
+            pcm_data, sample_rate = read_wave(temp_wav_path)
+            print(f"  [转换完成] 采样率: {sample_rate} Hz, 原始时长: {len(pcm_data) / (sample_rate * 2):.2f}s")
+    
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)